stdlib/benchmarks/collections/bench_string.mojo

```diff
@@ -261,30 +261,30 @@ def main():
             alias old = old_chars.get[j, StringLiteral]()
             alias new = new_chars.get[j, StringLiteral]()
             suffix = "[" + str(length) + "]"  # "(" + fname + ")"
-            m.bench_function[bench_string_count[length, fname, old]](
-                BenchId("bench_string_count" + suffix)
-            )
+            # m.bench_function[bench_string_count[length, fname, old]](
```

Comment:
**Question** Did you mean to leave these commented out?

Reply:
oops :laughing: